shared/markdown/parser.pegjs

```diff
@@ -73,7 +73,7 @@ Text
 
 // TODO: should coalesce multiple line quotes
```

Comment:
but given what we have, will it matter? we'll change the code and it'll look the same